CMBLensingSPT3GInterface/src/CMBLensingSPT3GInterface.jl

```diff
@@ -208,15 +223,15 @@ end
 """
 Convert a CMBLensing FlatS02 to a 3G MapSpectraTEB.
 """
-function MapSpectraTEB(f)
-    frame = Frame(Fourier(f), "TEB")
+function MapSpectraTEB(f::FlatIEBFourier)
+    frame = Frame(f, "TEB")
     py"MapSpectraTEB($frame)"o
 end
 
 ### py -> jl
 ############
 function Base.convert(::Type{FlatIEBFourier}, f::PyObject)
-    @assert pytypeof(f) == py"MapSpectraTEB "
```

Comment:
I've found that without this trailing space, Atom's syntax highlighting incorrectly reads `py"...B"` as a an unclosed `b"...` `Base.CodeUnits` string and ruins highlighting for the rest of the document. I may add these spaces back in unless you know of a workaround?

Reply:
Which versions of Atom stuff are you on? Its working ok for me: 

![image](https://user-images.githubusercontent.com/1510968/88489605-f9478e80-cf49-11ea-97e2-700d371c3a04.png)